packages/shorebird_cli/lib/src/commands/init_command.dart

```diff
@@ -30,81 +35,59 @@ class InitCommand extends ShorebirdCommand
       return ExitCode.noUser.code;
     }
 
-    final progress = logger.progress('Initializing Shorebird');
     try {
       if (!hasPubspecYaml) {
-        progress.fail('''
+        logger.err('''
 Could not find a "pubspec.yaml".
 Please make sure you are running "shorebird init" from the root of your Flutter project.
 ''');
         return ExitCode.noInput.code;
       }
     } catch (error) {
-      progress.fail('Error parsing "pubspec.yaml": $error');
+      logger.err('Error parsing "pubspec.yaml": $error');
       return ExitCode.software.code;
     }
 
-    final ShorebirdYaml? shorebirdYaml;
-    try {
-      shorebirdYaml = getShorebirdYaml();
-    } catch (_) {
-      progress.fail('Error parsing "shorebird.yaml".');
+    if (hasShorebirdYaml) {
+      logger.err('''
+"shorebird.yaml" already exists.
+Please remove it and try again.''');
       return ExitCode.software.code;
     }
 
-    String? appId;
-
-    if (shorebirdYaml != null) {
-      final codePushClient = buildCodePushClient(
-        httpClient: auth.client,
-        hostedUri: hostedUri,
-      );
-
-      final List<App> apps;
-      final fetchAppsProgress = logger.progress('Fetching apps');
-      try {
-        apps = (await codePushClient.getApps())
-            .map((a) => App(id: a.appId, displayName: a.displayName))
-            .toList();
-        fetchAppsProgress.complete();
-      } catch (error) {
-        fetchAppsProgress.fail('$error');
-        return ExitCode.software.code;
-      }
+    final Set<String> productFlavors;
+    try {
+      productFlavors = await extractProductFlavors(Directory.current.path);
+    } catch (error) {
+      logger.err('$error');
+      return ExitCode.software.code;
+    }
 
-      final app = apps.firstWhereOrNull(
-        (a) => a.id == shorebirdYaml!.appId.value,
+    final AppId appId;
+    try {
+      final displayName = logger.prompt(
+        '${lightGreen.wrap('?')} How should we refer to this app?',
```

Comment:
Should we standardize the green "?" at the beginning of prompts/confirms? I've noticed this in a few places and haven't been sure whether I should do this as well.

Reply:
Yeah I can do that in a subsequent PR maybe an extension on `Logger` called `confirm`?

Reply:
That sounds great, I was just asking so to make sure my prompts are consistent :)